sql/moz-fx-data-shared-prod/telemetry_derived/active_users_rollup_shredder_v1/schema.yaml

```diff
@@ -0,0 +1,53 @@
+fields:
+- name: submission_date
+  type: DATE
+  mode: NULLABLE
+  description: Submission date recorded from telemetry.active_users
+- name: app_name
+  type: STRING
+  mode: NULLABLE
+  description: App name recorded from telemetry.active_users
+- name: logical_dag_date
```

Comment:
This is smart - to separate the logical date (the parameter the job was triggered with) from the actual date the query was run